packages/tradershub/src/constants/constants.tsx

```diff
@@ -10,12 +10,8 @@ import {
     CurrencyUsdIcon,
     CurrencyUsdtIcon,
 } from '@deriv/quill-icons';
+import { PlatformIcon } from '../components';
```

Comment:
Is your Alias PR merged?
If yes then can we start using aliases?

Reply:
No, i dont want it to be merged till @sergei-deriv's Text PR is merged. So that he doesnt face conflicts

Reply:
ok